### DISCUSSION TITLE
LME v1.2.0

### DISCUSSION THREAD

**Discussion by mitchelbaker-cisa** [cisagov/LME#103] [Announcements]
## Added
- Added documentation instructions to upgrade from 1.1.0→1.2.0

## Changed
- Updated the Elastic Stack to v8.11.1 to fix the security vulnerability that was in earlier versions

## Fixed
- Deploy script should not time out anymore, we now pull Elasticsearch images before doing upgrade or install

## Notes
- Review [upgrading.md](https://github.com/cisagov/LME/blob/main/docs/markdown/maintenance/upgrading.md) for upgrade instructions

**Comment by OSHEAN** [comment]
Hi, not sure if this is the correct place to report this.  (LME v1.20) Fresh install of ELK performed twice (Ubuntu 22.04). Followed directions and  executed each command and then the install script.  Messages display that passwords are being Set but the newly created passwords aren't displayed when the install script finishes and as a result logging in isn't possible.  Please advise.

**Comment by llwaterhouse** [comment]
When you want to report a bug, please create a gitHub Issue and fill out the template so that we may best help you.
Thank you.
